MoltenVK/MoltenVK/GPUObjects/MVKDevice.mm

```diff
@@ -1075,6 +1110,24 @@
         _features.drawIndirectFirstInstance = true;
     }
 
+#if MVK_TVOS
+    _features.textureCompressionETC2 = true;
+    _features.textureCompressionASTC_LDR = true;
+
+	if (supportsMTLFeatureSet(tvOS_GPUFamily1_v3)) {
+		_features.dualSrcBlend = true;
+	}
+
+    if (supportsMTLFeatureSet(tvOS_GPUFamily2_v1)) {
+        _features.occlusionQueryPrecise = true;
+    }
+
+	if (supportsMTLFeatureSet(tvOS_GPUFamily2_v1)) {
+		_features.tessellationShader = true;
+		_features.shaderTessellationAndGeometryPointSize = true;
```

Comment:
It occurs to me that a) we don't support geometry shaders, and b) we don't support `PointsMode` on tessellation shaders. Therefore, we can't set the point size from either of these shaders. Setting this for the other platforms was a mistake on my part, which I need to correct.

Reply:
I think you're asking me to remove this line, which I have done.